tabs/configuration.js

```diff
@@ -456,6 +456,10 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
             serialRXtypes.push('Spektrum Bidir SRXL');
         }
 
+        if (semver.gte(CONFIG.flightControllerVersion, "3.2.0"))  {
```

Comment:
i totally disagree, the most transparent thing to do is check the flight controller version